rbx_binary/src/tests/serializer.rs

```diff
@@ -56,9 +56,10 @@ fn unknown_property() {
 /// This test will need to be updated once we implement the type used here.
 #[test]
 fn unimplemented_type_known_property() {
-    let tree = WeakDom::new(
-        InstanceBuilder::new("UIListLayout").with_property("Padding", UDim::new(1.0, -30)),
-    );
+    let tree = WeakDom::new(InstanceBuilder::new("UIListLayout").with_property(
+        "Padding",
+        Region3::new(Vector3::new(0.0, 0.0, 50.0), Vector3::new(0.0, 0.0, 50.0)),
+    ));
```

Comment:
This is kind of funny, can we pick a property that definitely doesn't exist here instead of one that does, but with the wrong type?